### ISSUE TITLE
Mind releasing the compiled .sfx output?

### ISSUE THREAD

**Issue by heetbeet** [git-for-windows/7-Zip#6]
Hi, I would like to use this sfx fork in one of my project, but am currently struggling to compile this through Visual Studio. Would you mind releasing the compiled binaries from this fork?

**Comment by heetbeet** [comment]
Never mind, I managed to strip it out myself using the following Python code:
```
header = bytearray.fromhex("377ABCAF271C")
header_config = b";!@Install@"

with open("PortableGit-2.32.0-rc1-64-bit.7z.exe", "br") as f:
    gbin = f.read()
    
sfx = gbin[:gbin.find(header)]
sfx_bin = sfx[:sfx.find(header_config)]
sfx_config = sfx[sfx.find(header_config):]

with open("git.sfx", "wb") as f: 
    f.write(sfx_bin)

with open("git.config", "wb") as f: 
    f.write(sfx_config)
```

Here are the results for anyone else wanting the same:
[git sfx.zip](https://github.com/git-for-windows/7-Zip/files/6560037/git.sfx.zip)

**Comment by dscho** [comment]
Hey, I'd love to release it officially. But I want something in return: a GitHub workflow that builds it. Could you take a crack at it?

**Comment by heetbeet** [comment]
Cool, I'm quite busy for the next two weeks or so, so I'll take a crack at it end of June :)

**Comment by dscho** [comment]
The end of June has passed. [I took a crack at it](https://github.com/git-for-windows/7-Zip/pull/8) and was done within an hour (read: it was not all that hard, and the burden should not have needed to rest on my shoulders), but I think this concludes the total time I will spend on this here ticket.

**Comment by heetbeet** [comment]
I'm sorry @dscho, my behavior on this thread is exactly the kind of thing that makes FOSS tedious and thankless.  I've been on the receiving end as well and it's no fun: I shouldn't have let this task become your problem. What I'll do in the future on an FOSS issue/forum like this is

- If I make a date or promise, add it to my work calendar
- Communicate my proficiency for a given task
- Provide proper feedback on my progress

Re the task, this was my first attempt at a GitHub action and wanted to learn to also use it in my own work. I first tried to compile 7-zip locally (failed with the latest Visual Studio, succeeded with the 2015 one IIRC ), then realize that _local_ doesn't really translate to cloud-actions via yaml config. I then started to see if I can get some of our company's python tests on GitHub actions (after hours), since I understand the Python tooling better than Visual Studio. I'm still in that process
